packages/components/src/utils/getAlertStatus/getAlertStatus.ts

```diff
@@ -0,0 +1,9 @@
+type CmpsrAlertStatus = 'success' | 'error' | 'warning' | 'inform';
+type ChakraAlertStatus = 'success' | 'error' | 'warning' | 'info';
+
+export const getAlertStatus = (status: ChakraAlertStatus): CmpsrAlertStatus =>
+  status === 'info' ? 'inform' : status;
```

Comment:
Could we fix the `Alert` to have `inform` instead of `info` so we can avoid this?

Reply:
I think we can do that if we create a copy of the the `useToast` chakra hook and override the params type to pass that value @guilean